scala/scala.bzl

```diff
@@ -928,12 +928,13 @@ _common_attrs_for_plugin_bootstrapping = {
   "print_compile_time": attr.bool(default=False, mandatory=False),
 }
 
-_common_attrs = _common_attrs_for_plugin_bootstrapping + {
+_common_attrs = dict(_common_attrs_for_plugin_bootstrapping)
```

Comment:
Why do you use dict() here but below (for example _scala_test_attrs you don't use it?